Softeq.XToolkit.WhiteLabel.iOS/Helpers/AvatarImageHelpers.cs

```diff
@@ -43,30 +43,32 @@ private static UIImage CreateCircleImage(Size size, CGColor backgroundColor, Act
             return image;
         }
 
-        private static void DrawText(string text, Size size, UIFont font)
+        private static void DrawText(string text, Size size, UIFont font, UIColor textColor)
         {
             var attributedText = text.BuildAttributedString()
                 .Font(font)
-                .Foreground(UIColor.White);
+                .Foreground(textColor);
 
             var textSize = attributedText.Size;
             var textPoint = new CGPoint(
-                size.Width / 2f - textSize.Width / 2f,
-                size.Height / 2f - textSize.Height / 2f);
+                (size.Width / 2f) - (textSize.Width / 2f),
+                (size.Height / 2f) - (textSize.Height / 2f));
             attributedText.DrawString(textPoint);
         }
 
         public class AvatarStyles
         {
-            public AvatarStyles(Size size, UIFont font, string[] backgroundHexColors)
-            {
-                Size = size;
-                Font = font;
-                BackgroundHexColors = backgroundHexColors;
+            public AvatarStyles(Size size, UIFont font, string[] backgroundHexColors)

```

Comment:
Wouldn't you want to pass TextColor to the constructor as well? (probably with default value for backward-compatibility)

Reply:
- `TextColor` is optional
- Default value is `White`